src/compiler/types.ts

```diff
@@ -3610,7 +3616,8 @@ namespace ts {
         JSX = 2,
         TS = 3,
         TSX = 4,
-        External = 5
+        External = 5,
+        JSON = 6

```

Comment:
Do we really need a `ScriptKind.JSON` if we don't have `JsonSourceFile` extend `SourceFile`?